### PR TITLE
fix/improve exception handling

### PR DESCRIPTION
**Pull request by ff137**
Stacktrace only prints when prod == False. prod is true by default

### CODE REVIEW

File: app/facades/__init__.py
Comment (wdbasson) [inline app/facades/__init__.py:1]: What are these empty files?

**Review by wdbasson — COMMENTED** [review]
@ff137 how do you set `prod == False`? Is it possible to do this with an env var?

File: app/facades/__init__.py
Comment (ff137) [reply app/facades/__init__.py:1]: Empty `__init__.py` files are necessary for the interpreter to recognize imports of modules in the directory. It's not strictly necessary in newer version of Python, but it's considered best practice & it's primarily for imports to resolve correctly in vscode

Comment (ff137) [reply app/facades/__init__.py:1]: with regards to setting `prod == False` , there seems to be an environment variable called `prod`. If it doesn't exist, defaults to true, as follows: `prod = strtobool(os.environ.get("prod", "True"))`.

If I search for "prod" it doesn't seem to be set in any of the docker files... so we should make a todo for that - either create new env var or point to a different one.

**Review by wdbasson — APPROVED** [review]
LGTM

**Comment by ff137** [comment]
Replicated from clean branch in https://github.com/didx-xyz/aries-cloudapi-python/pull/308

**Pull request closed without merging**